opencti-platform/opencti-graphql/src/modules/organization/organization.graphql

```diff
@@ -156,7 +156,7 @@ type OrganizationEdge {
 
 # Queries
 type Query {
-  organization(id: String!): Organization @auth(for: [KNOWLEDGE, SETTINGS_SETACCESSES])
+  organization(id: String!): Organization @auth(for: [KNOWLEDGE, SETTINGS_SETACCESSES, VIRTUAL_ORGANIZATION_ADMIN])
```

Comment:
This is really weird to imagine a user without KNOWLEDGE capa being admin of an organization 🤔 . It is possible for all user with KNOWLEDGE capa to query organization without SETTINGS_SETACCESSES or VIRTUAL_ORGANIZATION_ADMIN.

Reply:
> It is possible for all user with KNOWLEDGE capa to query organization without SETTINGS_SETACCESSES or VIRTUAL_ORGANIZATION_ADMIN.

Technically yes, even if it is meaningless functionally speaking.

I'm not fond of relying on a supposed user behavior, we better be explicit.
If we really want orga admin to have KNOWLEDGE or SETTINGS_SETACCESSES, we can force it / check it somehow. I'm not sure we want to go that path.